### PR TITLE
new page 

### PR DESCRIPTION
**Pull request by ndongsara**

### CODE REVIEW

File: Index.php [hunk 1]
```diff
@@ -62,19 +62,24 @@
...
-                                    echo substr($listeQuestion["question"], 0, 155)."...";
-                                } else {
-                                    echo ($listeQuestion["question"]);
-                            }?>
-                        </li>
+
+                        <li>
+                            <?php if (strlen($listeQuestion["question"]) > 155) { ?>
+                                    <?php echo $index;?>"> <?php echo substr($listeQuestion["question"], 0, 155)."..."; ?>
+                               <?php  } else { ?>
```
Comment (heidisbk) [inline Index.php]: useless php tags

File: Index.php [hunk 2]
```diff
@@ -62,19 +62,24 @@
...
+                            <a class="link" href="Question_page.php?id=<?php echo $index;?>"> <?php echo ($listeQuestion["question"]);?> </a>
```
Comment (heidisbk) [inline Index.php]: it would be wise to make the whole div clickable

File: Assets/css/Question_form.css [hunk 3]
```diff
@@ -109,6 +109,50 @@ h2 {
     display: inline-table;
 }
 
+
+
+.answ{
```
Comment (gabfour) [inline Assets/css/Question_form.css]: Even though the names are different, please avoid using two names that similar

File: Index.php [hunk 4]
```diff
@@ -62,19 +62,24 @@
...
+                            <?php }?>
+                            </li>
+                        <p><em><?php echo "Nombre de réponses: $answerCount"; ?></em></p>
```
Comment (gabfour) [inline Index.php]: Useless, already done in a better way line 76-80

File: Assets/css/Question_form.css (hunk 3 above)
Comment (gabfour) [reply Assets/css/Question_form.css]: edit: talking about .answ and .answer

**Pull request merged**